modules/custom-post-types/portfolios.php

```diff
@@ -713,6 +723,19 @@ static function get_project_tags( $post_id ) {
 	}
 
 	/**
+	 * Displays the author of the current portfolio project.
+	 *
+	 * @return html
+	 */
+	static function get_project_author() {
+		$html = '<div class="project-author"><span>' . esc_html__( 'Author:', 'jetpack' ) . '</span> ';
```

Comment:
Thanks for looking into this @zinigor, what do you think about the snippet I posted in my previous comment?